SonarQube.MSBuild.Tasks/AttachBuildWrapper.cs

```diff
@@ -68,39 +73,32 @@ public AttachBuildWrapper(int buildWrapperTimeoutInMs)
         public override bool Execute()
         {
             Debug.Assert(!string.IsNullOrWhiteSpace(this.BinDirectoryPath), "Expecting the BinDirectoryPath to have been set");
-            Debug.Assert(!string.IsNullOrWhiteSpace(this.OutputDirectoryPath), "Expecting the OutputDirectoryPath to have been set");
+            Debug.Assert(!string.IsNullOrWhiteSpace(this.AnalysisConfigDir), "Expecting the AnalysisConfigDir to have been set");
 
-            bool success = CheckRequiredFilesExist();
-            if (!success)
+            if (!CheckRequiredBinariesExist())
             {
                 return false;
             }
 
-            try
```

Comment:
The analysis on Dory complained about the try-catch block not being tested. On looking at the code again, we aren't expecting any unhandled exceptions so I removed it. We are launching the monitor as an external process, so we only need to look at the return code to see if it succeeded or not. We won't be affected by any exceptions thrown in the external process.